synapse/__init__.py

```diff
@@ -36,6 +37,14 @@
 except ImportError:
     pass
 
+# Use the standard library json implementation instead of simplejson.
+try:
+    from canonicaljson import set_json_library
+
+    set_json_library(json)
+except ImportError:
```

Comment:
I guess this means we don't *need* to bump the canonicaljson version since the import will error, but it seems good to require the new version.

Reply:
out of interest, why do we have this guard?

Reply:
This code runs during a call to `setup.py` in order to get the version information. (This is also why we have this guard around the twisted imports above.)

Reply:
aha!

Reply:
I've added a comment (above the twisted imports) about this! Hopefully this means someone else won't have to discover this in the future. 😄